packages/backend/src/graphql/mutations/update-flow-status.ts

```diff
@@ -43,24 +42,10 @@ const updateFlowStatus = async (
     pattern: interval || EVERY_15_MINUTES_CRON,
   }
 
-  if (trigger.type === 'webhook') {
-    const $ = await globalVariable({
-      flow,
-      connection: await triggerStep.$relatedQuery('connection'),
-      app: await triggerStep.getApp(),
-      step: triggerStep,
-      testRun: false,
-    })
-
-    if (flow.active && trigger.registerHook) {
-      await trigger.registerHook($)
-    } else if (!flow.active && trigger.unregisterHook) {
-      await trigger.unregisterHook($)
-    }
-  } else {
+  if (trigger.type !== 'webhook') {
     if (flow.active) {
       flow = await flow.$query().patchAndFetch({
```

Comment:
Out of scope but just a comment - I don't see why we can't update `publishedAt` for webhook pipes. Will make a PR to change this later.